src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -341,13 +341,13 @@
     <value>Symbols should be compared for equality, not identity. Use an overload accepting an 'IEqualityComparer' and pass 'SymbolEqualityComparer'.</value>
```

Comment:
The existing description (not touched in this PR) doesn't make much sense to me for the same reasoning in https://github.com/dotnet/roslyn-analyzers/pull/5535#r717964748.

Reply:
Equality kind of stretches into inclusion/exclusion of nullable here. Shorthand to say "Should compare for equality" makes some sense to me, but I definitely see where the ", not identity" can be misleading.

Reply:
> Equality kind of stretches into inclusion/exclusion of nullable here


That's exactly it. this is what should be indicated by the diagnostic.

Probably:

- Title: `Make your symbol comparison intent explicit`
- Message: `Specify whether you want to include nullability when comparing symbols`
- Description: `When comparing symbols, explicitly specify a 'SymbolEqualityComparer' to indicate whether you want to include nullability in comparison`

(I know my wording isn't very good, but something along those lines?)

Reply:
What about: 

* Title: `Symbol comparison should use 'SymbolEqualityComparer'`
* Message: `Use a 'SymbolEqualityComparer' to specify whether to include nullability when comparing symbols`
* Description: `When comparing symbols, it's good practice to specify whether symbols with different nullability should be considered equal. Use a 'SymbolEqualityComparer' to specify in this comparison whether  to include nullability as part of the equality check.`

Reply:
That way the title tells you what we expect, the message and description both elaborate on the _why_

Reply:
Much better!

Reply:
Thanks for the feedback :) I'll see if I can get that in this afternoon